test/basic-querying.test.js

```diff
@@ -1030,6 +1037,7 @@ describe('basic-querying', function() {
   });
 });
 
+// FIXME: This should either be re-enabled or removed.
```

Comment:
I wasn't going to re-enable this test as a part of this PR since it's unrelated.
I just want to mark which tests are selectively enabled via support flags for clarity.